fe1-web/src/features/lao/navigation/LaoNavigation.tsx

```diff
@@ -51,49 +52,51 @@ const LaoNavigation: React.FC = () => {
   }, [passedScreens]);
 
   return (
-    <OrganizationTopTabNavigator.Navigator
-      initialRouteName={STRINGS.navigation_lao_home}
-      screenOptions={{
-        tabBarActiveTintColor: Color.accent,
-        tabBarInactiveTintColor: Color.inactive,
-        headerLeftContainerStyle: {
-          paddingLeft: Spacing.contentSpacing,
-        },
-        headerRightContainerStyle: {
-          paddingRight: Spacing.contentSpacing,
-        },
-        headerTitleStyle: Typography.topNavigationHeading,
-        headerTitleAlign: 'center',
-      }}>
-      {screens.map(
-        ({
-          id,
-          title,
-          headerTitle,
-          Component,
-          headerShown,
-          headerLeft,
-          headerRight,
-          tabBarIcon,
-        }) => (
-          <OrganizationTopTabNavigator.Screen
-            key={id}
-            name={id}
-            component={Component}
-            options={{
-              title: title || id,
-              headerTitle: headerTitle || title || id,
-              headerLeft,
-              headerRight,
-              tabBarIcon: tabBarIcon || undefined,
-              // hide the item if tabBarIcon is set to null
-              tabBarItemStyle: tabBarIcon === null ? { display: 'none' } : undefined,
-              headerShown,
-            }}
-          />
-        ),
-      )}
-    </OrganizationTopTabNavigator.Navigator>
+    <NoCurrentLaoErrorBoundary>
```

Comment:
So if an error happens there, the whole component will be replaced by the simple text (in `NoCurrentLaoErrorBoundary.tsx`) saying that you're not connected to a LAO. Am I right?

Reply:
Exactly :) But yes I think you're right, I should add some documentation to the file :)

Reply:
Unfortunately at the moment during development the error still triggers a popup but the UI does not break and the error should not show up in production. The reason is that it somehow is caught but then somehow still "thrown". I am not yet sure how to prevent this since this is an error we expect to happen.

Reply:
Actually it seems to be intended this way 😅 https://stackoverflow.com/a/47400249/2897827